content/_help/manage-your-account/resolve-duplicate-account._en.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: help
+title: How do I fix duplicate accounts?
+category: manage-your-account
+permalink: /help/manage-your-account/resolve-duplicate-account/
+order: 10
+---
+Some government agencies only allow you to have one verified Login.gov account. If you have more than one Login.gov account using the same verified information, you’ll need to delete the duplicate accounts.
+
+{: .number-list}
+1. **Choose an account to keep**
+    Keep the account you use the most. For example, the one tied to the most services. That way, you won’t have to reconnect services.
+
+    To see which agencies an account is connected to, sign in and click on the “Your connected accounts” in the left sidebar.
+
+2. **Delete the duplicate accounts**
+    To [delete an account](/help/manage-your-account/delete-your-account/):
+    * Sign into the account you are going to delete.
+    * Go to “Your account” and choose “Delete your account” in the left sidebar.
+
+    If you [forgot your password](/help/trouble-signing-in/forgot-your-password/), try resetting it from the sign in page.
+
+    If you can’t authenticate into the duplicate account:
+    * Click “Choose another method” when signing in.
+    * Scroll to the bottom of the page and select “Delete your account”.
+
+3. **Sign in with your remaining account**
+    Go back to the agency’s website and sign in using the one Login.gov account you kept.
+
+
+## What if I don’t recognize or can’t delete an account?
+If you don’t recognize or can’t delete one of the duplicate accounts you saw on the multiple accounts page, call our contact center at (844) 875-6446 and provide them with the error code LG33.
```

Comment:
```suggestion
If you don’t recognize or can’t delete one of the duplicate accounts you saw on the multiple accounts page, call our contact center at **(844) 875-6446** and provide them with the error code **LG33**.
```
Following the design that I see.

Reply:
Do we follow the convention of having a non-breaking space in the phone number?

Reply:
@vrajmohan Good question, I suppose like character 0xA0? That seems nice to have.

Reply:
The phone number would be **(844)&nbsp;875-6446**, or, even better, **(844)&nbsp;875&#8209;6446**.